docs/howto.md

```diff
@@ -83,27 +373,214 @@ The child account must have an administrative IAM Role with a trust relationship
 }
 ```
 
-Add the account to the pool using the DCE API:
+Add the account to the pool using the DCE API
+
+**Request**
+
+`POST ${api_url}/accounts`
```

Comment:
It would be great if we could somehow provide [swagger ui links](https://petstore.swagger.io/?_ga=2.251168133.206317326.1577721714-338265257.1569482972#/pet/uploadFile) rather than manually updating these every time something changes.